src/test/kotlin/org/rust/ide/annotator/RsErrorAnnotatorTest.kt

```diff
@@ -966,6 +967,22 @@ class RsErrorAnnotatorTest : RsAnnotatorTestBase() {
         }
     """)
 
+    fun `test type mismatch E0308 coerce reference to ptr`() = expect<ComparisonFailure> {
```

Comment:
Hmm, if the test currently not passes let's add some TODO comment describes it

Reply:
Done